client/components/accordion/test/index.jsx

```diff
@@ -114,5 +114,23 @@ describe( 'Accordion', () => {
 			expect( toggleSpy ).toHaveBeenCalledWith( false );
 			expect( wrapper.state( 'isExpanded' ) ).toBe( false );
 		} );
+
+		test( 'should render as expected with a title and content when initially open', () => {
+			const wrapper = shallow(
+				<Accordion initialExpanded={ true } title="Section">
+					Content
+				</Accordion>
+			);
+
+			expect( wrapper.hasClass( 'accordion' ) ).toBe( true );
+			expect( wrapper.state( 'isExpanded' ) ).toBe( true );
+			expect( wrapper.hasClass( 'has-icon' ) ).toBe( false );
+			expect( wrapper.hasClass( 'has-subtitle' ) ).toBe( false );
+			expect( wrapper.find( '.accordion__icon' ) ).toHaveLength( 0 );
+			expect( wrapper.find( '.accordion__title' ).text() ).toBe( 'Section' );
+			expect( wrapper.find( '.accordion__subtitle' ) ).toHaveLength( 0 );
+			expect( wrapper.find( '.accordion__icon' ) ).toHaveLength( 0 );
+			expect( wrapper.find( '.accordion__content' ).text() ).toBe( 'Content' );
```

Comment:
Are all these elements checks really relevant to the tested functionality? Maybe we could make the test more minimal.

Reply:
Nope, they're not really needed for this functionality, and the test at the top already does all these checks. Let me make that change, then.